openfisca_us/variables/expense/spm_unit_federal_tax.py

```diff
@@ -7,3 +7,6 @@ class spm_unit_federal_tax(Variable):
     label = "SPM unit federal tax"
```

Comment:
Also for FICA while you're here please.
```suggestion
    label = "Federal income tax"
```